benches/benches/benchmarks/util.rs

```diff
@@ -213,7 +213,7 @@ lazy_static! {
         let script = Script::new_builder()
             .code_hash(CellOutput::calc_data_hash(&data))
             .args(vec![Bytes::from(PUBKEY_HASH.as_bytes()).pack()].pack())
-            .hash_type(ScriptHashType::Data.pack())
+            .hash_type(ScriptHashType::Data as u8)
```

Comment:
Expose interfaces is better than expose inner structures.

We can put `as u8` in `pack()` and let `pack()` be `inline`.

Reply:
Agreed, we can unify the interface.

Reply:
I prefer `std::convert::Into` to `Pack` trait,  updated code, please review again, thanks

Reply:
1. I can **NOT** change `Into` but I can change `Pack`.
2. When put `JsonScriptHashType` and `ScriptHashType` together, it's a bit difficult to distinguish between them (both have `Into`).
3. `hash_type` is `u8`, it's an implementation detail, it's an optimization.
   In a complete logic, it's should be `molecule::Byte(bytes::Bytes)` (`it.as_slice().len() == 1`).
   So, make it be special, when I update internal implementation, I have to update all things here.